wework/scripts/dev-mac-app.sh

```
@@ -156,23 +186,41 @@ require_command git
+if [ "${#DEV_IDENTITY_FIELDS[@]}" -ne 6 ]; then
+  echo "Error: failed to resolve the development instance identity." >&2
+  exit 1
+fi
+export WEWORK_PARENT_TITLE="${DEV_IDENTITY_FIELDS[0]}"
+export WEWORK_DEV_TITLE="${DEV_IDENTITY_FIELDS[1]}"
+export WEWORK_DEV_INSTANCE_ID="${DEV_IDENTITY_FIELDS[2]}"
+export WEWORK_DEV_INSTANCE_LABEL="${DEV_IDENTITY_FIELDS[3]}"
+export WEWORK_DEV_DOCK_TITLE="${DEV_IDENTITY_FIELDS[4]}"
+export WEWORK_DEV_EXECUTABLE_NAME="${DEV_IDENTITY_FIELDS[5]}"
+export WEWORK_APP_IDENTIFIER="${WEWORK_DEV_APP_IDENTIFIER:-io.wecode.wework.dev.$WEWORK_DEV_INSTANCE_ID}"
 export WEWORK_USER_DATA_DIR="$(
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)